package.json

```diff
@@ -10,23 +10,25 @@
     "lint:fix": "next lint --fix"
   },
   "dependencies": {
-    "@headlessui/react": "^1.4.2",
-    "axios": "^0.21.1",
+    "@headlessui/react": "^1.7.19",
+    "axios": "^0.21.4",
     "next": "^14.2.3",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "next-themes": "^0.3.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-icons": "^5.2.1",
     "swr": "^1.3.0"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.12.1",
-    "@babel/preset-react": "^7.18.6",
+    "@babel/eslint-parser": "^7.24.6",
+    "@babel/preset-react": "^7.24.6",
     "@next/eslint-plugin-next": "^14.2.3",
-    "@tailwindcss/forms": "^0.5.2",
-    "autoprefixer": "^10.4.2",
-    "eslint": "^8.27.0",
-    "eslint-config-next": "^14.0.3",
+    "@tailwindcss/forms": "^0.5.7",
+    "autoprefixer": "^10.4.19",
+    "eslint": "^8.57.0",
+    "eslint-config-next": "^14.2.3",
```

Comment:
`eslint-config-next` should not be added here, `@next/eslint-plugin-next` is already present, checkout this PR where I explain this: https://github.com/laravel/breeze-next/pull/49